packages/terra-collapsible-menu-view/src/CollapsibleMenuView.jsx

```diff
@@ -158,6 +181,7 @@ class CollapsibleMenuView extends React.Component {
       { 'is-calculating': this.isCalculating },
       { 'collapsible-menu-view-flex-end': !isStartAligned },
       { 'collapsible-menu-view-flex-start': isStartAligned },
+      { reverse: isReversed },
```

Comment:
I noticed in testing the example that the tab order of these items is reversed (i.e. hitting Tab goes from right to left). Will this be addressed in a future change?

Reply:
The order will be reversed again in `terra-filepath`, I tried to not add another set of logic within here to not clutter it with filepath code

Reply:
I didn't catch that, thanks. I didn't realize flex-direction also affected tab order :(